lib/utilityservice.js

```diff
@@ -874,6 +886,18 @@ UtilityService.prototype.sfdx = async function(command, options) {
         } else if (command == 'mdapi:listmetadata') {
             var MdapiListmetadataCommand = require("salesforce-alm/dist/commands/force/mdapi/listmetadata");
             sfdxCommand = new MdapiListmetadataCommand.MdapiListmetadataCommand();
+        } else if (command == 'source:convert') {
+            var SourceConvertCommand = require("salesforce-alm/dist/commands/force/source/convert");
+            flags.quiet = true;
```

Comment:
For these `flags.` you can remove since they are set above. The one for 'org:display' is always set to quiet because it can display a token that shouldn't be there in the logs.

Reply:
both `flags.quiet` and `flags.json` were removed